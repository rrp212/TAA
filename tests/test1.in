50
empty
size
insert 100
empty
size
insert 200
insert 300
insert 400
empty
size
clear
empty
size
insert 100
count 100
remove 100
empty
count 100
insert 100
insert 200
insert 300
insert 400
insert 500
count 100
count 400
count 500
lower_bound 0
lower_bound 1000
lower_bound 100
lower_bound 350
lower_bound 500
upper_bound 0
upper_bound 1000
upper_bound 100
upper_bound 350
upper_bound 500
range_search 0 1000
range_search 50 150
range_search 100 100
range_search 350 370
range_search 420 730
range_search 500 500
range_search 0 10
range_search 1000 1010
min
max
remove 100
remove 500
min
max
