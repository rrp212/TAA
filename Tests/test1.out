true
0
insert 100
false
1
insert 200
insert 300
insert 400
false
4
clear
true
0
insert 100
true
remove 100
true
false
insert 100
insert 200
insert 300
insert 400
insert 500
true
true
true
100
false
100
400
500
100
false
200
400
false
true
true
true
false
true
true
false
false
100
500
remove 100
remove 500
200
400